File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-1/java/ProbeTests.java
// TEST-1 probe, Java. Exactly 3 tests using JUnit 5 (org.junit.jupiter), the test
// framework admissible for Java under the three-clause tool admissibility test:
// two assert a true condition, one asserts a false condition.
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class ProbeTests {
    @Test
    void passOne() { assertTrue(Probe.add(1, 1) == 2); }

    @Test
    void passTwo() { assertTrue(Probe.add(2, 3) == 5); }

    @Test
    void failOne() { assertTrue(Probe.add(1, 1) == 3); }
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-1/kotlin/ProbeTests.kt
// TEST-1 probe, Kotlin. Exactly 3 tests using kotlin.test (kotlin-test.jar ships in
// the kotlinc distribution): two assert a true condition, one asserts a false condition.
import kotlin.test.Test
import kotlin.test.assertTrue

class ProbeTests {
    @Test
    fun passOne() { assertTrue(add(1, 1) == 2) }

    @Test
    fun passTwo() { assertTrue(add(2, 3) == 5) }

    @Test
    fun failOne() { assertTrue(add(1, 1) == 3) }
}
